spoon-client/src/main/java/com/squareup/spoon/Spoon.java

```diff
@@ -90,6 +103,22 @@ public static File screenshot(Activity activity, String tag, String testClassNam
     }
   }
 
+  /**
+   * Call this method to set the screenshots compression format and compression quality.
+   * @param compressionFormat The format of the compressed image (JPEG, PNG or WEBP)
+   * @param compressionQuality int: Hint to the compressor, 0-100.
+   *        0 meaning compress for small size,
+   *        100 meaning compress for max quality.
+ *          Some formats, like PNG which is lossless, will ignore the quality setting
+   */
+  public static void setCompressionFormat(final Bitmap.CompressFormat compressionFormat,
+     final int compressionQuality) {
+    if (compressionFormat != null) {
```

Comment:
instead of ignoring null, let's do
```
if (compressionFormat == null) {
  throw new IllegalArgumentException("Null is not a valid CompressFormat");
}
```